code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -62,10 +62,8 @@
 		strengthdiv = 8
 		for(var/mob/living/simple_animal/revenant/R in get_hearers_in_view(7,get_turf(holder.my_atom)))
 			var/deity
-			if(SSreligion.deity)
-				deity = SSreligion.deity
-			else
-				deity = "Christ"
+			if(GLOB.deity)
```

Comment:
why do you have the check if you are adding a `||` later

Reply:
woops!